app/app.module.ts

```diff
@@ -0,0 +1,27 @@
+﻿/// <reference path="../typings/browser.d.ts" />
+'use strict';
+namespace dogsrus.virtdog {
+  (() => {
+    angular.module('app', [
```

Comment:
This is a personal preference, but here you are defining all your code within the same namespace. I would use a different namespace for each feature module, for example dogsrus.virtdog.dog, etc. That way the exported types are only implicitly available within the same module, and explicitly via the module name.